src/main/java/org/mitre/synthea/helpers/RandomCodeGenerator.java

```diff
@@ -1,191 +1,111 @@
 package org.mitre.synthea.helpers;
 
-import ca.uhn.fhir.context.FhirContext;
-import com.google.common.cache.CacheBuilder;
-import com.google.common.cache.CacheLoader;
-import com.google.common.cache.LoadingCache;
-import java.util.Objects;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 import java.util.Random;
-import java.util.concurrent.ExecutionException;
+
 import javax.annotation.Nonnull;
-import org.hl7.fhir.r4.model.IntegerType;
-import org.hl7.fhir.r4.model.UriType;
-import org.hl7.fhir.r4.model.ValueSet;
-import org.hl7.fhir.r4.model.ValueSet.ValueSetExpansionComponent;
-import org.hl7.fhir.r4.model.ValueSet.ValueSetExpansionContainsComponent;
+
+import org.apache.commons.lang3.StringUtils;
 import org.mitre.synthea.world.concepts.HealthRecord.Code;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
 
 /**
- * Generates random codes based upon ValueSet URIs, with the help of a FHIR terminology service
- * API.
+ * Generates random codes based upon ValueSet URIs, with the help of a FHIR
+ * terminology service API.
```

Comment:
Why change the formatting of this comment?

Reply:
Its resolved.